CMakeLists.txt

```diff
@@ -68,15 +68,22 @@ set(OMR_INSTALL_DATA_DIR ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME} CACHE PATH "In
 ###
 
 if (OMR_ENV_DATA64)
-	find_package(Threads)
+	find_package(Threads QUIET)
 	if(Threads_FOUND)
 		# Threads mush be found before we can look for CUDA.
 		# FindCuda will error out if Threads is missing, even though CUDA itself is optional.
-		find_package(CUDA)
+		find_package(CUDA QUIET)
+		if (CUDA_FOUND)
+			message(STATUS "OMR: CUDA enabled")
```

Comment:
If someone requests to explicitly disable CUDA support, will this code still run (and print "OMR: CUDA enabled" if all the libraries are present)?

Reply:
Yep. Want that changed?

Reply:
If CUDA is explicitly disabled, there should be no messages about any attempts to find it.

Reply:
On the flip side, if CUDA is explicitly enabled and directed to use a specific version, this should not fail if `nvcc` is not found (OMR doesn't need it). See [1] which only makes the necessary header files available.

[1] https://github.com/eclipse/openj9/pull/7280

Reply:
OK, is there a specific header I can check for in the build system? And I suppose this means there are _no_ CUDA libraries we need to link? On what platforms do we support CUDA?

Reply:
OMR uses few of the header files from CUDA: I think checking for `include/cuda.h` is probably sufficient.